conbench/entities/distribution.py

```diff
@@ -17,7 +17,7 @@ class Distribution(Base, EntityMixin):
     case_id = NotNull(s.String(50), s.ForeignKey("case.id", ondelete="CASCADE"))
     context_id = NotNull(s.String(50), s.ForeignKey("context.id", ondelete="CASCADE"))
     commit_id = NotNull(s.String(50), s.ForeignKey("commit.id", ondelete="CASCADE"))
-    machine_hash = NotNull(s.String(250))
+    hardware_hash = NotNull("machine_hash", s.String(250))
```

Comment:
I'm a bit surprised by `"machine_hash"` here. Should this be `"hardware_hash"`? Or are we keeping the `machine_hash` name around for some of these bits? I'm sure there's a good reason — I just can't figure it out so far

Reply:
This is a SQLAlchemy way of allowing me to rename all fields in code (`machine_hash` to `hardware_hash`, `machine` to `hardware`) without actually renaming this field in the actual database.

Renaming machine table and all its foreign keys, indexes, constraints at the db level will need to be handled in a separate migration and PR. I am still deciding if I should do this though since this will be giant. It is doable though.

Reply:
Aaah I see! Yeah I agree on pushing that migration out if we can. Maybe a comment here explaining that would be helpful to remind us that's what's going on?

Reply:
Added comments in 3 places where `machine_hash` comes up.